lib/membrane_vpx/encoder/vpx_encoder.ex

```diff
@@ -128,17 +132,18 @@ defmodule Membrane.VPx.Encoder do
         {fixed_deadline, _framerate} -> fixed_deadline |> Membrane.Time.as_microseconds(:round)
       end
 
-    rc_target_bitrate = get_target_bitrate(state.rc_target_bitrate, width, height, framerate)
+    user_encoder_config =
+      process_user_encoder_config(state.user_encoder_config, width, height, framerate)
```

Comment:
Couldn't we store the already processed `user_encoder_config` in the element's state?

Reply:
we could, but the target bitrate needs to be recalculated on new stream format either way, so i don't think there's much difference